app/src/main/java/gov/hhs/cdc/trustedintermediary/external/reportstream/ReportStreamLabOrderSender.java

```diff
@@ -42,14 +42,13 @@ public void sendOrder(final LabOrder<?> order) {
         sendRequestBody(json, bearerToken);
     }
 
-    protected String sendRequestBody(
-            @NotNull String json, @NotNull String bearerToken) { // url param?
+    protected String sendRequestBody(@NotNull String json, @NotNull String bearerToken) {
         String res = "";
         Map<String, String> headers =
                 Map.of(
                         "Authorization", "Bearer " + bearerToken,
                         "client", "flexion",
```

Comment:
The `client` should be `flexion.etor-service-sender`.

Reply:
Good catch!